deploy/run_with_summon.sh

```diff
@@ -5,39 +5,44 @@ set -xeuo pipefail
 
 # Clean up when script completes and fails
 function finish {
-  announce 'Wrapping up and removing test environment'
-
   # There is a TRAP in test_in_docker.sh to account for Docker deployments so we do not need to add another one here
   # Stop the running processes
-  if [[ $RUN_IN_DOCKER = false ]]; then
+   if [[ $RUN_IN_DOCKER = false && $DEV = false ]]; then
+    announce 'Wrapping up and removing dev environment'
     ./stop
-    ../kubernetes-conjur-deploy-"$UNIQUE_TEST_ID"/stop
+    cd ./kubernetes-conjur-deploy-$UNIQUE_TEST_ID && ./stop
     # Remove the deploy directory
     rm -rf "../kubernetes-conjur-deploy-$UNIQUE_TEST_ID"
-  fi
-
+   fi
 }
 trap finish EXIT
 
-./platform_login.sh
+if [ "${DEV}" = "false" ]; then
+  ./platform_login.sh
+fi
 
 ./1_check_dependencies.sh
 
-./stop
```

Comment:
why don't we stop anymore? shouldn;t it be conditional?

Reply:
it stales the local dev so wrapping it in conditional